Python/Twitter_Images_Downloader/README.md

```diff
@@ -0,0 +1,21 @@
+## Twitter Images Downloader
+
+This script downloads images from given user's recent tweets and also creates a CSV file with `Tweet URL`, `Image ID`, `Image URL` columns.
+
+### Instructions
+
```

Comment:
Just add another instruction to install the requirements for running the script

Reply:
Also, in the requirements.txt file, specify the versions of your packages.

Reply:
Done